packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountWelcomeScreen.js

```diff
@@ -1,156 +1,164 @@
 import s from '../../../common/locales/strings.js'
 import * as Constants from '../../../constants/index.js'
 import * as Styles from '../../../styles/index.js'
 import { type Branding } from '../../../types/Branding.js'
 import { type Dispatch, type RootState } from '../../../types/ReduxTypes.js'
 import { logEvent } from '../../../util/analytics.js'
-import { scale } from '../../../util/scaling.js'
-import { ImageHeaderComponent } from '../../abSpecific/ImageHeaderComponent'
-import { Button } from '../../common/Button.js'
-import T from '../../common/FormattedText.js'
-import { HeaderBackButton } from '../../common/HeaderBackButton.js'
-import SafeAreaView from '../../common/SafeAreaView.js'
 import { connect } from '../../services/ReduxStore.js'
+import {
+  type Theme,
+  type ThemeProps,
+  withTheme
+} from '../../services/ThemeContext'
+import { EdgeText } from '../../themed/EdgeText'
+import { SimpleSceneHeader } from '../../themed/SimpleSceneHeader'
+import { SecondaryButton } from '../../themed/ThemedButtons'
+import { ThemedScene } from '../../themed/ThemedScene'
 
 type OwnProps = {
   branding: Branding
 }
 type DispatchProps = {
-  onBack(): void,
   onDone(): void
 }
-type Props = OwnProps & DispatchProps
+type Props = OwnProps & DispatchProps & ThemeProps
 
-type State = {}
+const NewAccountWelcomeScreenComponent = ({
+  theme,
+  branding,
+  onDone
+}: Props) => {
+  const styles = getStyles(theme)
+  const appName = branding.appName || s.strings.app_name_default
 
-class NewAccountWelcomeScreenComponent extends React.Component<Props, State> {
-  render() {
-    return (
-      <SafeAreaView>
-        <View style={styles.screen}>
-          <View style={styles.row1}>
-            <HeaderBackButton
-              testID="exitButton"
-              onPress={this.props.onBack}
-              styles={styles.exitBackButtonStyle}
-              label={s.strings.exit}
-            />
+  return (
+    <ThemedScene>
+      <SimpleSceneHeader>{s.strings.get_started}</SimpleSceneHeader>
+      <View style={styles.content}>
+        <Image source={LOGO_BIG} style={styles.logo} resizeMode="contain" />
+        <EdgeText style={styles.welcome}>
+          {sprintf(s.strings.welcome, appName)}
+        </EdgeText>
+        <View style={styles.advantage}>
+          <Image
+            source={WELCOME_LOCK}
+            style={styles.advantageImage}
+            resizeMode="contain"
+          />
+          <View style={styles.advantageTextContainer}>
+            <EdgeText style={styles.advantageTitle}>
+              {sprintf(s.strings.welcome_advantage_one_title, appName)}
+            </EdgeText>
+            <EdgeText style={styles.advantageDescription}>
+              {s.strings.welcome_advantage_one_description_1}
+            </EdgeText>
+            <EdgeText style={styles.advantageDescription}>
+              {s.strings.welcome_advantage_one_description_2}
+            </EdgeText>
```

Comment:
This two lines shouldn't be 2 separate text component.
It can look really off on different size screens or different translations.

They should be joined into one sentence with auto wrapping.